iree/compiler/Conversion/Common/LinalgBufferizePass.cpp

```diff
@@ -1089,6 +1130,15 @@ static LogicalResult convertVectorTransferWriteOp(OpBuilder &b,
   return success();
 }
 
+static LogicalResult convertScfForOp(OpBuilder &b, scf::ForOp forOp,
+                                     BlockAndValueMapping &bvm,
+                                     BufferizationPlan &plan) {
+  Value result = forOp.results()[0];
```

Comment:
This is assuming a single `init` Arg. I think this should be done for all init-args to handle multiple result case.

Reply:
this was updated in a latter commit